cmd/gotohelm/main.go

```diff
@@ -5,27 +5,42 @@ import (
 	"fmt"
 	"os"
 	"path"
+	"strings"
 
 	"github.com/redpanda-data/helm-charts/pkg/gotohelm"
 	"golang.org/x/tools/go/packages"
 )
 
+type dependencies []string
+
+func (d *dependencies) String() string {
+	return fmt.Sprint(*d)
+}
+
+func (d *dependencies) Set(value string) error {
+	for _, dep := range strings.Split(value, ",") {
+		*d = append(*d, strings.TrimSpace(dep))
+	}
+	return nil
+}
+
 func main() {
+	var deps dependencies
 	out := flag.String("write", "-", "The directory to write the transpiled templates to or - to write them to standard out")
-
+	flag.Var(&deps, "deps", "The list, separated by comma, of dependencies which will be transpiled")
 	flag.Parse()
 
 	cwd, _ := os.Getwd()
 
 	pkgs, err := gotohelm.LoadPackages(&packages.Config{
 		Dir: cwd,
-	}, flag.Args()...)
+	}, deps, flag.Args()...)
```

Comment:
As noted in `LoadPackages`, couldn't you instead call `LoadPackages(..., append(flag.Args(), deps...)...)` here?

Reply:
I will do that, thanks for suggestion.